tools/setup/emoji/build_emoji

```diff
@@ -240,6 +240,62 @@ def setup_emoji_farm(cache_path, emoji_map, emoji_data):
 
     generate_sprite_css_files(cache_path, emoji_map, emoji_data)
 
+def setup_old_emoji_farm(cache_path, emoji_map):
+    # type: (Text, Dict[Text, Text]) -> None
+    # Code for setting up old emoji farm.
+    remapped_emojis = {
+        "0023": "0023-20e3",    # Hash
+        "0030": "0030-20e3",    # Zero
+        "0031": "0031-20e3",    # One
+        "0032": "0032-20e3",    # Two
+        "0033": "0033-20e3",    # Three
+        "0034": "0034-20e3",    # Four
+        "0035": "0035-20e3",    # Five
+        "0036": "0036-20e3",    # Six
+        "0037": "0037-20e3",    # Seven
+        "0038": "0038-20e3",    # Eight
+        "0039": "0039-20e3",    # Nine
+        "1f48f": "1f469-200d-2764-200d-1f48b-200d-1f468",    # Couple kiss
+        "1f491": "1f469-200d-2764-200d-1f468",  # Couple with heart
+    }
+
+    os.chdir(cache_path)
+    cache_emoji = os.path.join(cache_path, 'images', 'emoji')
+    cache_emoji_unicode = os.path.join(cache_path, 'images', 'emoji', 'unicode')
+    cache_google_64 = os.path.join(cache_path, 'images-google-64')
```

Comment:
Maybe these should be called:

`emoji_cache_path`, `unicode_emoji_cache_path`, `google_emoji_cache_path`.

I like to include e.g. `_path` at the end of the name for variables that are paths since it makes the code more readable; I was confused for a bit reading the below about where `cache_emoji_unicode` was set because it sounded like it was the unicode codepoint or something :)